.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: test
+on: [push, pull_request]
+jobs:
+
+  shellcheck:
+    name: "Run shellcheck on the shell env script"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: "Install shellcheck"
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y shellcheck
+    - name: "Run shellcheck on the shell env script"
+      run: |
+        # exclude warning for sourcing missing external file
+        shellcheck --exclude=SC1091 extra/share/dune/env/env.sh extra/share/dune/env/env.bash
```

Comment:
Should the warning be excluded rather in the script itself, to also spare devs the noise locally? Or is there a reason for a disparity between the dev environment and what the CI sees?